103.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+from collections import deque
+
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def zigzagLevelOrder(self, root: TreeNode) -> List[List[int]]:
+        levels = []
+        
+        if not root:
+            return levels
+        
+        queue = deque([root])
+        toggle = False
+        
+        while queue:
+            level = []
```

Comment:
As you can see, using this template we can make `level` local to the `while queue:` loop, whereas in your [previous template](https://github.com/woke7/leetcode-python/pull/17/commits/15d960af88781ca13ede8aaf9ddd41a820c7db15) you had to declare `level` outside of the loop. This is a good example of making your code less fragmented.